UnitTests/PayPalTests/MockCheckout.swift

```diff
@@ -3,14 +3,15 @@ import UIKit
 
 @testable import PayPalNativeCheckout
 @testable import PaymentsCore
+import PayPalCheckout
```

Comment:
Do we need this import?

Reply:
yeah `ApprovalOrderIntent` is from this

Reply:
For NXO classes the approach we have been taking is to wrap the types vs pulling them in directly. Is there a specific reason we needed to move to use `ApprovalOrderIntent` vs using the String as we were previously?